src/usersyncroniser.ts

```diff
@@ -233,10 +233,10 @@ export class UserSyncroniser {
         let mxidExtra = "";
         if (webhookID) {
```

Comment:
if we are no longer using the ID, can this be a boolean instead?

Reply:
done